### ISSUE TITLE
Unable to build entry_table example

### ISSUE THREAD

**Issue by lgrapenthin** [janestreet/incr_dom#20]
(I'm more interested in running the examples than building them, but they don't seem to be hosted anywhere (?)

This is mostly, because incremental is often referenced as more or less the defacto standard in incremental web ui calculation, but it appears to be impossible to find a single running web ui that runs on it. )

```
[user@dev incr_dom]$ dune build --verbose example/entry_table/{main.bc.js,index.html}
Shared cache: disabled
Shared cache location: /home/user/.cache/dune/db
Workspace root: /home/user/Documents/projects/incremental/incr_dom
Auto-detected concurrency: 2
Dune context:
 { name = "default"
 ; kind = "default"
 ; profile = Dev
 ; merlin = true
 ; fdo_target_exe = None
 ; build_dir = In_build_dir "default"
 ; instrument_with = []
 }
Actual targets:                   
- _build/default/example/entry_table/index.html
Error: Don't know how to build example/entry_table/main.bc.js
```

Somebody else mentioned that it would be useful to add "js" to the respective dune files "modes" list, so I tried this.

Then I get this

```
File "src/js_misc.ml", line 35, characters 35-45:
35 |   Float.( >= ) (Js.float_of_number rect##.top) 0.
                                        ^^^^^^^^^^
Error: This expression has type float but an expression was expected of type
         Js.number Js.t
File "src/start_app.ml", line 500, characters 2-50:
500 |   Inline_css.Private.update_if_not_already_updated ();
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error: Unbound value Inline_css.Private.update_if_not_already_updated
```

**Comment by TyOverby** [comment]
If you're trying to build the examples off of a github checkout, then you'll need to use all the jane street libraries pinned to github too

**Comment by lgrapenthin** [comment]
Since I didn't know how to do this exactly I went back and built incr_dom from opam again (instead this repo). For this I had to "unpin" this repo / which for some reason was pinned... Then I went back in commits until it got to a point where the examples were compatible with the released version.
You might want to close this issue, because I apparently don't understand enough about Ocaml / its build tooling to provide you a useful problem report. I hoped that following the instructions of the example readme would work, but apparently more experience with Ocaml tooling is required.
